components/tools/ToolsList.js

```diff
@@ -6,7 +6,7 @@ export default function toolsList({ toolsData }) {
     <div className="" data-testid="ToolsList-main" >
       {Object.keys(toolsData).map((categoryName, index) => {
         if(toolsData[categoryName].toolsList.length > 0) return (
-        <div className='my-8' key={index} id={categoryName}>
+        <div className='my-8 scroll-m-96 scroll-mb-96' key={index} id={categoryName}>
```

Comment:
Why there is a change in the design ??

Reply:
When we select the category from the dropdown directly, it takes us to the heading of that category, but it does not happen if we directly enter the URL. Hence this change in the CSS.